runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -25,6 +25,7 @@
 #include "tflite_loader.h"
 #include "json/json.h"
 #include "ir/OpCode.h"
+#include "ir/Models.h"
```

Comment:
Can I refer to this header for asynchronous run feature in progress ?

Reply:
No problem if you need this for draft or test. But this is also WIP, so it may be changed frequently.